pkg/storaged/stratis-details.jsx

```diff
@@ -442,20 +456,27 @@ export const StratisPoolDetails = ({ client, pool }) => {
                 </CardTitle>
             </CardHeader>
             <CardBody>
-                <DescriptionList className="pf-m-horizontal-on-sm">
-                    <DescriptionListGroup>
-                        <DescriptionListTerm className="control-DescriptionListTerm">{_("storage", "UUID")}</DescriptionListTerm>
-                        <DescriptionListDescription>{ pool.Uuid }</DescriptionListDescription>
-                    </DescriptionListGroup>
-                    { use &&
-                    <DescriptionListGroup>
-                        <DescriptionListTerm className="control-DescriptionListTerm">{_("storage", "Usage")}</DescriptionListTerm>
-                        <DescriptionListDescription className="pf-u-align-self-center">
-                            <StorageUsageBar stats={use} critical={0.95} />
-                        </DescriptionListDescription>
-                    </DescriptionListGroup>
-                    }
-                </DescriptionList>
+                <Stack hasGutter>
+                    {alerts.length && alerts.map((alert, idx) => (
```

Comment:
This produces a bigus "0" when `alerts` is empty:

![image](https://github.com/cockpit-project/cockpit/assets/3228183/f76d097f-186d-4189-87b6-dc4a29962bf0)